File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.java
class Solution {
    public int[] searchRange(int[] nums, int target) {
        int[] list=new int[]{-1,-1};
        for(int i=0;i<nums.length;i++)
        {
            if(nums[i]==target)
            {
                if(list[0]==-1)
                    list[0]=i;
                list[1]=i;
            }
        }
        return list;
    }
}

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.py
class Solution(object):
    def searchRange(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        st,end=-1,-1
        for i in range(len(nums)):
            if nums[i]==target:
                if st==-1:
                    st=i
                end=i
        return [st,end]        